internal/commands/app/diff.go

```diff
@@ -14,6 +14,16 @@ import (
 	"github.com/AlecAivazis/survey/v2"
 )
 
+const (
+	flagIncludeNodeModules  = "include-node-modules"
+	flagIncludePackageJSON  = "include-package-json"
+	flagIncludeDependencies = "include-dependencies"
+)
+
+const (
+	errDependencyFlagConflictTemplate = `cannot use both "%s" and "%s" at the same time`
```

Comment:
[style nit] could avoid the backticks here and use `%q` instead of `"%s"` (no action needed here though)